MH4U Dex/AreaDropTableViewCell.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AreaDropTableViewCell.h
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/11/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+#import "AreaDrop.h"
```

Comment:
So would that look something like:


#import "AreaDrop.m"

@class AreaDrop;